src/app/profile/cleanup/cleanup.component.html

```diff
@@ -9,7 +9,7 @@ <h3 *ngIf="cleanupStatus === 'notstarted'">
         <strong>You are about to erase all of your OpenShift.io spaces</strong>, reverting your OpenShift.io environment to it's default state.
```

Comment:
Do we still want this to call out "spaces". This line is bold. The changed one is less noticeable.